src/routes/boot.ts

```diff
@@ -394,6 +393,8 @@ const loggedInBoot = async (
         // read only, used in frontend to decide if banner should be fetched
         banner:
           lastBanner !== 'false' && alerts.lastBanner < new Date(lastBanner),
+        // read only, used in frontend to decide if boot popup should be shown
+        bootPopup: !isSameDay(alerts.lastBootPopup, new Date()),
```

Comment:
Don't think it matters as it gets stored in server time and read in server time.
It will always match set day

Reply:
But don't we send the lastBootPopup date from apps? https://github.com/dailydotdev/apps/blob/5300036d1bcefda60b62207c27f8a858f2419d25/packages/shared/src/components/modals/useBootPopups.tsx#L166

Maybe I'm confusing it..

Reply:
Ah yeah I actually had this as isoString before.
We can also consider making it a endpoint like the other alert that sets the date on server?

It's way safer as we don't have to deal with timezones.
But since it's a day to day thing I really don't think we'll reach much concerns here.
Happy to see what others think.

Reply:
> Ah yeah I actually had this as isoString before.

Maybe we can do that again. 

> But since it's a day to day thing I really don't think we'll reach much concerns here.

True don't think there will be much concerns, maximum we'll be showing 2 popups per day per some users, it's just about how we want to go about this fix, are we ok to allow it for some edge cases with timezones, or we want to bullet proof it.

Reply:
> We can also consider making it a endpoint like the other alert that sets the date on server?

Where did we solve it like that?

Reply:
With all timezone problems lately I would also prefer if we just keep it server side 😆

Reply:
@capJavert updateLastReferralReminder does it for instance it's a seperate call.
Let me know if you want me to use that :)

Reply:
@nensidosari @capJavert modified to handle SS date